platform-ajax/src/test/java/com/softicar/platform/ajax/testing/selenium/engine/common/AjaxSeleniumTestServerWaiterTest.java

```diff
@@ -0,0 +1,84 @@
+package com.softicar.platform.ajax.testing.selenium.engine.common;
+
+import com.softicar.platform.ajax.testing.selenium.engine.level.low.AbstractAjaxSeleniumLowLevelTest;
+import com.softicar.platform.dom.elements.DomDiv;
+import com.softicar.platform.dom.event.IDomBlurEventHandler;
+import com.softicar.platform.dom.event.IDomClickEventHandler;
+import com.softicar.platform.dom.event.IDomEvent;
+import com.softicar.platform.dom.event.IDomFocusEventHandler;
+import com.softicar.platform.dom.event.IDomInputEventHandler;
+import com.softicar.platform.dom.input.DomTextInput;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Test;
+
+public class AjaxSeleniumTestServerWaiterTest extends AbstractAjaxSeleniumLowLevelTest {
+
+	private final TestDiv testDiv;
+	private final List<String> events;
+
+	public AjaxSeleniumTestServerWaiterTest() {
+
+		this.testDiv = openTestNode(TestDiv::new);
+		this.events = new ArrayList<>();
+	}
+
+	@Test
+	public void test() {
+
+		send(testDiv.input1, "a");
+		click(testDiv.input2);
+		waitForServer();
+
+		assertEquals("[FOCUS#1, INPUT#1, BLUR#1, FOCUS#2, CLICK#2]", events.toString());
```

Comment:
I don't think the test setup and this assertion in particular are sufficient. With a certain amount of luck, this assertion could pass even without the `waitForServer` call.
Could we somehow provoke failure, such that this assertion is highly likely to fail if the `waitForServer` call would be omitted? e.g. via introduction of an artificial delay

Reply:
did so